scripts/rollback_awslambda_image.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+REPO="pcluster-manager-awslambda"
```

Comment:
thought(non-blocking): why not adding usage instruction in a readme, and then convert it to a runbook? at least we have something to start with :)

Reply:
ok will do